Test/FMI3/fmi3_capi_basic_test.cpp

```diff
@@ -78,6 +78,51 @@ static void test_int_get_set(fmi3_import_t* fmu, jm_status_enu_t instantiate_sta
         REQUIRE(int64_values[1] == values_from_int_get[1]);
 }
 
+/* Helper function to verify get/set for float64 works as expected.
+    The input arguments are:
+        the FMU to use get/set on, note that this has to be the DUMMY FMU (of any kind)
+        the returned value from the FMU instantiation that has be done before calling this function.
+*/
+static void test_real_get_set(fmi3_import_t* fmu, jm_status_enu_t instantiate_status){
+        size_t n_value_references = 4;
+        /* This value reference does not matter because the Dummy FMU
+            implementation for set/get int64 uses a fixed dummy array.
+        */
+        size_t n_float_values = 7;
+        /* Sizes in arrays below are explicitly hard-coded to confine to older gcc standards */
+        const fmi3_value_reference_t value_references[4] = {0, 1, 3, 12};
+        const fmi3_float64_t grav = 4.;
```

Comment:
I couldnt figure out why it was named to `grav`, is it related to `inst->states[VAR_R_HEIGHT_SPEED] = 4;`?